### PR TITLE
LPS-156454 Update integration tests

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java [hunk 1]
```diff
@@ -224,7 +224,7 @@ public ObjectDefinition addOrUpdateSystemObjectDefinition(
 					newObjectField.getDBType(),
 					newObjectField.getDefaultValue(), false, false, "",
 					newObjectField.getLabelMap(), newObjectField.getName(),
-					newObjectField.isRequired());
+					newObjectField.isRequired(), newObjectField.getState());
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java:227]: Hi @seltong,

Could we replace `newObjectField.getState()`to `newObjectField.isState()`?

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java [hunk 2]
```diff
@@ -723,9 +725,16 @@ private void _validateDefaultValue(String businessType, String defaultValue)
...
+		if (Objects.equals(
 				ObjectFieldConstants.BUSINESS_TYPE_PICKLIST, businessType)) {
 
+			if (!state) {
+				throw new ObjectFieldStateException(
+					"Object Field default value can only be added when the " +
+						"state is true");
+			}
+		}
+		else {
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java:737]: Hi @seltong,

What do you think about this approach below?

````
private void _validateDefaultValue(
			String businessType, String defaultValue, boolean state)
		throws PortalException {

		if (Validator.isNull(defaultValue)) {
			return;
		}

		if (!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-152677"))) {
			throw new UnsupportedOperationException();
		}

		if (!Objects.equals(
				ObjectFieldConstants.BUSINESS_TYPE_PICKLIST, businessType)) {

			throw new ObjectFieldDefaultValueException(
				"Object field with different business type from picklist " +
					"must not have default type.");
		}

		if (!state) {
			throw new ObjectFieldStateException(
				"Object field default value can only be added when the " +
					"picklist is a state.");
		}
	}

````

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:object

**Comment by seltong** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Pull request closed without merging**